cmake/faceliftMacros.cmake

```diff
@@ -379,13 +381,28 @@ macro(_facelift_parse_target_arguments additionalOptions additionalOneValueArgs
     endforeach()
 
     set(HEADERS ${ARGUMENT_HEADERS})
-    foreach(HEADER_GLOB ${ARGUMENT_HEADERS_GLOB_RECURSE})
-        file(GLOB_RECURSE GLOB_FILES ${HEADER_GLOB})
-        list(APPEND HEADERS ${GLOB_FILES})
+
+    set(HEADERS_NO_MOC ${ARGUMENT_HEADERS_NO_MOC})
+
+    # files for installation CANNOT be in both sections: HEADERS and HEADERS_NO_MOC
+    foreach(HEADER ${HEADERS})
```

Comment:
the same restriction applies to HEADERS and HEADERS_NO_INSTALL, and we are not checking it.